src/App.test.tsx

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import { shallow, ShallowWrapper } from 'enzyme';
 
-it('renders without crashing', () => {
-  shallow(<App />);
+import { UnconnectedApp } from './App';
+import { anonymousUser } from './store/auth/types';
+import { Homepage } from './pages/Homepage';
+import { Footer } from './components/Footer';
+
+describe('<UnconnectedApp />', () => {
+  const mockHandleInitialData = jest.fn();
+  const wrapper = shallow(
+    <UnconnectedApp handleInitialData={mockHandleInitialData} />
+  );
+  it('should chontain Homepage component and Footer component', () => {
+    expect(wrapper.find(Homepage)).toHaveLength(1);
+    expect(wrapper.find(Footer)).toHaveLength(1);
```

Comment:
Alternatively this is a good use case for [snapshot testing](https://jestjs.io/docs/en/snapshot-testing)